config/locales/en.yml

```diff
@@ -29,5 +29,26 @@
 # To learn more, please read the Rails Internationalization guide
 # available at http://guides.rubyonrails.org/i18n.html.
 
+
+# This is a yaml file that is loaded by rails, it lets you use translate your
+# define texts in a different language. In our case we are using it to customise
+# our messages for our validation checks
 en:
-  hello: "Hello world"
+  activerecord:
+    errors:
+      format: "%{message}"
+      models:
+        user:
+          attributes:
+            email:
+              blank: "Please enter an email address"
```

Comment:
low priority: in our agreed effort to make a bona fide facebook clone, shall we make sure the error messages reflect what facebook actually does as closely as possible?